src/common/constants.ts

```diff
@@ -25,6 +25,8 @@ export const domains = [
     'deriv-com-git-fork',
```

Comment:
Please refactor this strings in a variable and use it here

Reply:
okay